flutter_web_auth_2/lib/flutter_web_auth_2.dart

```diff
@@ -10,6 +10,7 @@ export 'src/options.dart';
 export 'src/unsupported.dart'
     if (dart.library.io) 'src/server.dart'
     if (dart.library.html) 'src/web.dart';
+export 'src/windows.dart';
```

Comment:
I believe this breaks the Web implementation, but I need to test properly. If it really breaks web, then we should really think about Linux support using the `desktop_webview_window` approach.